<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>443</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Электросила</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>2</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="PrepareForm" name="tabInputParameters">
       <attribute name="title">
        <string>Автоматическая проверка</string>
       </attribute>
      </widget>
      <widget class="Measurements" name="tabMeasurements">
       <attribute name="title">
        <string>Ручное управление</string>
       </attribute>
      </widget>
      <widget class="Communications" name="tabCommunications">
       <attribute name="title">
        <string>Связь</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>857</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PrepareForm</class>
   <extends>QWidget</extends>
   <header>autoform.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Measurements</class>
   <extends>QWidget</extends>
   <header>measurements.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Communications</class>
   <extends>QWidget</extends>
   <header>communications.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
